bio/minibwa/map/wrapper.py

```
@@ -0,0 +1,154 @@
+        # Add sort order
+        if sort_order == "queryname":
+            samtools_sort_opts += " -n "
+
+        # Add user-defined sort arguments
+        samtools_sort_opts += f" {sort_extra}"
+
+        out_cmd = f" | samtools sort {samtools_sort_opts} -T '{tempdir}'"
+
+    elif sort == "picard":
+        # Get java optional parameters
+        java_opts = get_java_opts(snakemake)
```

Ruff (0.15.21)

[error] Undefined name `snakemake`

(F821)